configuration/reports/reportdescriptors/dataexports/allergies.yml

```diff
@@ -25,3 +25,4 @@ config:
   components:
     - "allergies"
     - "allDataExports"
+  privilege: "Task: clinical.reports"
```

Comment:
Not really a big deal, but since we are creating the variable "${privilege.task_lab_reports}", we could use that in all the yml files instead of hardcoding "Task: clinical.reports"